tock/tock/utils.py

```diff
@@ -3,14 +3,18 @@
 import os
 import sys
 
-from django.core.exceptions import PermissionDenied
+from django.contrib.auth.mixins import LoginRequiredMixin
 from rest_framework.permissions import BasePermission
+from django.shortcuts import redirect
+
+# from uaa_client.authentication import UaaBackend
```

Comment:
Should this commented-out line of code be removed?

Reply:
Good catch, most likely unless it'll be needed again. :-)